models/issue.go

```diff
@@ -148,6 +148,19 @@ func (issue *Issue) loadAssignee(e Engine) (err error) {
 	return
 }
 
+func (issue *Issue) loadPullRequest(e Engine) (err error) {
+	if issue.IsPull && issue.PullRequest == nil {
+		issue.PullRequest, err = getPullRequestByIssueID(e, issue.ID)
+		if err != nil {
+			if IsErrPullRequestNotExist(err) {
+				return err
+			}
+			return fmt.Errorf("getPullRequestByIssueID [%d]: %v", issue.ID, err)
```

Comment:
I can change it if you really want, but this is more consistent with the other `loadXYZ(..)` methods.